app/routes/bdb/BdbDetailRoute.js

```diff
@@ -35,23 +48,40 @@ const mapStateToProps = (state, props) => {
   const companyEvents = selectEventsForCompany(state, { companyId });
   const companySemesters = selectCompanySemesters(state, props);
   const fetching = state.companies.fetching;
+  const showFetchMoreEvents = selectPagination('events', {
+    queryString: queryString(companyId)
+  })(state);
   return {
     company,
     companyId,
     companyEvents,
     comments,
     companySemesters,
-    fetching
+    fetching,
+    showFetchMoreEvents
   };
 };
 
-const mapDispatchToProps = {
-  deleteSemesterStatus,
-  deleteCompanyContact,
-  editSemesterStatus,
-  editCompany,
-  deleteCompany,
-  deleteComment
+const mapDispatchToProps = (dispatch, props) => {
+  const { companyId } = props.params;
+  const fetchMoreEvents = () =>
+    dispatch(
+      fetchEventsForCompany({
+        queryString: queryString(companyId),
+        loadNextPage: true
+      })
+    );
+  return {
+    deleteSemesterStatus: (companyId, semesterStatusId) =>
```

Comment:
10/10 code duplication. Looks kinda wanky too. :man_shrugging:  You can use `foo: (...args) => dispatch(foo(...args)`, or even better use `bindActionCreators`.